packages/website/blog/2024-05-27-announcing-typescript-eslint-v8-beta.mdx

```diff
@@ -311,13 +307,85 @@ createRule({
 
 See [feat(utils): allow specifying additional rule meta.docs in RuleCreator](https://github.com/typescript-eslint/typescript-eslint/pull/9025) for more details.
 
+### Flat configuration `RuleTester`
```

Comment:
```suggestion
### Flat Configuration `RuleTester`
```

I am sorry that I am like this 🫠

Reply:
![Cartoon capybaras playing in a hot tub. Caption: "ONE OF US! ONE OF US!"](https://github.com/user-attachments/assets/257be046-d011-4af2-b1e5-58a29736b05b)